NuGetPackages/readme/MonoGame.Framework/readme.txt

```diff
@@ -50,8 +52,8 @@ Finally, set the "Build Action" of the MGCB file to "MonoGameContentReference" (
 (Don't forget to exclude your existing .XNB / content files once you have upgraded)
 
 _Release Notes
-3.2.3 release
-Updated to latest development build (10th December 2014) Which includes the new 8.1 builds.
+3.3.0 release
```

Comment:
Do we need to include a change log here?  Can we not just redirect them to the website for all release notes?

My issue is that we don't want to have any manual stuff to change in the code base to make a release.